### ISSUE TITLE
Issue with shrinkage options in DESeq2 from DEBrowser

### ISSUE THREAD

**Issue by sriaim** [UMMS-Biocore/debrowser#220]
Hello, 

I was trying to run DE analysis with DESeq2 from DEBrowser (v1.21.1), but it keeps generating errors as shown below whenever I select any type of shrinkage methods (apeglm, ashr, or normal) with Wald test: 

Warning: Error in h: error in evaluating the argument 'x' in selecting a method for function 'nrow': subscript contains invalid names
  2: runApp
  1: startDEBrowser
Warning: Error in h: error in evaluating the argument 'x' in selecting a method for function 'nrow': subscript contains invalid names
  49: <Anonymous>

Could you please explain why this is happening? 

Thanks!

**Comment by nephantes** [comment]
make sure the delimiters are consistent in the file. If you chose tsv, it has to be tab-separated everywhere. If there is space somewhere it won't work. If you attach some portion of your data, I can help you to find the problem in your table.

**Comment by sriaim** [comment]
I used the Demo (Vernia et. al) to explore the software and it was directly loaded by clicking "Load Demo (Vernia et. al).

**Comment by nephantes** [comment]
Then there's something wrong about your installations. I suggest removing DEBrowser first. Restart your rstudio and reinstall it. It should work after that.

**Comment by sriaim** [comment]
I tried what you suggested, but still the same errors are occurring. Also, I noticed that when I deleted the DEBrowser and reinstalled after restarting the Rstudio, DEBrower version went down to 1.20.1.

**Comment by nephantes** [comment]
You can upgrade your BiocManager version to Release 3.14 to install the latest version to test.

**Comment by sriaim** [comment]
I have upgraded my BiocManager version to 3.14 and DEBrowser is now 1.22.1. 

if (!require("BiocManager", quietly = TRUE))
    install.packages("BiocManager")
BiocManager::install(version = "3.14")

However, I still see the error mentioned above, whenever I run Wald test with shrinkage methods. (The DE analysis has been working perfectly when shrinkage=None.)
I've tested multiple combinations, and found shrinkage methods only worked when LRT test type was selected with betaPrior=false. Is shrinkage only designed for LRT test or is it a type of technical error?

Thanks!
